Automattic/woocommerce-payments
# Update - Allow viewing Stripe account details only in page Overview.

#### Changes proposed in this Pull Request

When working on #2476, I found out users can access their Stripe Connect accounts in two places: 

- Payments > Overview - React (WcAdmin based).
- Payments > Settings (traditional WooCommerce > Settings page).

Then this PR does: 

- ~Redirect users to page Overview from page Settings when they want to edit their Stripe Express account.~
- Remove the Account section in the traditional WooCommerce > Settings page.
- Clean up the current code that is handling this action from two places. 
- ~Change the instruction text.~

Benefits: 

- Code is simplified and easier to maintain. 
- Consistent experience for users as they have only one place to access their Stripe info.









| Before  | After |
| ------------- | ------------- |
| ![Markup on 2021-07-20 at 19:24:08](https://user-images.githubusercontent.com/10045087/126323479-ab3d9e66-2919-49e5-9778-e83375ba0ad5.png)  | ![Markup on 2021-07-27 at 18:41:12](https://user-images.githubusercontent.com/10045087/127147831-cb823376-b5c4-496d-b68e-1e74068a1663.png) |


#### Testing instructions

- Apply this PR. 
- Visit http://site.com/wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments
- Expect: The `Account` section no longer exists. 

**If we have a consensus on this PR, I'd like to have more feedback regarding the wording change**. 






-------------------
